pkg/schemas/mapper/os_info.go

```diff
@@ -15,16 +16,16 @@ func (o OSInfo) FromInternal(data map[string]interface{}) {
 	if data == nil {
 		return
 	}
-	cpuInfo := map[string]interface{}{
-		"count": values.GetValueN(data, "capacity", "cpu"),
+	cpuInfo := map[string]interface{}{}
+	cpuNum, err := resource.ParseQuantity(convert.ToString(values.GetValueN(data, "capacity", "cpu")))
+	if err == nil {
+		cpuInfo["count"] = cpuNum.Value()
 	}
```

Comment:
Can we fall back on the original handling when the parse failed?

Reply:
Use `resource.ParseQuantity` can cover the original handling. 

before, we can handle values
memory: `Ki`
Cpu:  `“”`

Now we can handle all valid values in Kubernetes
memory: `Ki`, `Mi`, `Gi`, `Ti`, `Pi`, `Ei`
Cpu: `n`, `u`, `m`, `“”`, `k`, `M`, `G`, `T`, `P`, `E`